docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md

```diff
@@ -428,7 +428,7 @@ Note that the values set in the runtime environment always override the values d
 If you are exploring Docker and .NET Core on sources on the Internet, you will find Dockerfiles that demonstrate the simplicity of building a Docker image by copying your source into a container. These examples suggest that by using a simple configuration, you can have a Docker image with the environment packaged with your application. The following example shows a simple Dockerfile in this vein.
 
 ```Dockerfile
-FROM microsoft/dotnet
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1
```

Comment:
microservices guide is already 2.2 so if we have to reference a version use 2.2

However, since this example doesn't reference a version, we can just do the same

```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk
```

Reply:
Yeah @MichaelSimons, I think you're right, it should then be 2.2! at least for a while 😉